AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -58,6 +58,8 @@ siberianhealth.com##.top-informer-container
 gazetesok.com##.alert
 findagrave.com###Banner_cookie_0
 pamyra.de##app-cookie-banner
+xm.com#$#body { overflow: visible!important; }
+xm.com##.modal-backdrop
```

Comment:
all popup rules must be CSS, if you change style

Reply:
@Alex-302 done, take a look please